guides/source/configuring.md

```diff
@@ -954,6 +954,14 @@ text/javascript image/svg+xml application/postscript application/x-shockwave-fla
 
 * `config.active_storage.draw_routes` can be used to toggle Active Storage route generation. The default is `true`.
 
+* `config.active_storage.resolve_model_to_route` can be used to globally change how Active Storage files are delivered.
```

Comment:
@fleck FYI this is rendering a bit weird:

![image](https://user-images.githubusercontent.com/509837/103036127-9c6af700-452e-11eb-818f-ee219fff7cf0.png)

https://guides.rubyonrails.org/configuring.html#configuring-active-storage

Reply:
https://github.com/rails/rails/pull/40920 should fix it